### ISSUE TITLE
Attention: etc.ethermine.org/ has shut down on the 1st of July (01.07.2023). Consider staking on ethpool for an environmental-friendly alternative!

### ISSUE THREAD

**Issue by i3eaa** [UnamSanctam/SilentCryptoMiner#1869]
Hi so I was setting a miner up with the ETH classic pool with ethermine, it seems that the site had shut down presuming the header on July the 7th. I may be mistaken but is this right? Or is the pool still up and functionable. Also if you would recommend any other pools that would get good rates from people with GPUS/CPUS meant for gaming such as fortnite. Thank you!

**Comment by UnamSanctam** [comment]
Yes ethermine has stopped all their mining operations, you can instead use pools like 2miners or nanopool.

**Comment by i3eaa** [comment]
ok thank you!
